uktrade/great-cms
# Remove market guides feature flags

This PR removes two feature flags related to the market guide pages:
- `FEATURE_SHOW_MARKET_GUIDE_BAU_LINKS`
- `FEATURE_SHOW_MARKET_GUIDE_VISUALISATIONS`

These features are now 'on' permanently, so we no longer need any logic to hide them.

**To test locally**
- Make sure you don't have either feature flag in your secrets file
- Go to a market guide page e.g. `/markets/china`
- You should see the 3 stats boxes at the top of the page, and the list of related services on the right (assuming you have them populated in the CMS)

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-126

### Reviewing help


### Housekeeping


### Merging
